introduction/A_overview.md

```diff
@@ -56,16 +56,16 @@ Have a look at the [Plug Guide](http://www.phoenixframework.org/docs/understandi
 
 ### Ecto
 
-[Ecto](http://hexdocs.pm/ecto) is a language integrated query and database wrapper in Elixir. With Ecto, you can read and write to different databases, model your domain data, write complex queries in a type safe way, protecting you from SQL injection and other attacks and more.
+[Ecto](http://hexdocs.pm/ecto) is a language integrated query composition tool and database wrapper for Elixir. With Ecto, we can read and write to different databases, model our domain data, write complex queries in a type-safe way, protect against attack vectors - including SQL injection, and much more.
 
-Ecto exists around four main abstractions:
+Ecto is built around four main abstractions:
 
-* Repo - the repository is your storage. Every operations against the database is done via the repository
+* Repo - A repository represents a connection to an individual database. Every database operation is done via the repository.
 
-* Model - the model is your data definition. It defines the table name, fields and their respective types
+* Model - Models are our data definitions. They define table names and fields as well as each field's type.
 
-* Query - queries tie both models and repositories together, allowing us to elegantly retrieve data from the repository and cast it into the models themselves
+* Query - Queries tie both models and repositories together, allowing us to elegantly retrieve data from the repository and cast it into the models themselves.
 
-* Changeset - changesets are used to declare all changes to be performed against a model, providing type casting, validations and more
+* Changeset - Changesets declare all the changes we need to perform on our model data before our application can use it. These changes include type casting, validations, and more.
```

Comment:
"These changes include" -> "Changesets support" because we have something called changes and it is a map of values.